packages/client/lib/miner/miner.ts

```diff
@@ -216,7 +217,13 @@ export class Miner {
       const [signerAddress, signerPrivKey] = this.config.accounts[0]
       cliqueSigner = signerPrivKey
       // Determine if signer is INTURN (2) or NOTURN (1)
-      inTurn = await vmCopy.blockchain.cliqueSignerInTurn(signerAddress)
+      // console.log(vmCopy.blockchain)
+      // inTurn = await (vmCopy.blockchain.consensus as CliqueConsensus).cliqueSignerInTurn(
+      //   signerAddress
+      // )
```

Comment:
```suggestion
```

Reply:
Good catch. Removed this yesterday but missed it in today's reconciliation